source/docs/appendices/migration-guides.md

```diff
@@ -35,27 +52,322 @@ In this release, we have improved the image resizing mechanism to be bit smarter
 };
 ```
 
+## Added requisition list features to B2B module (only theme chocolatine suported)
+
+If you are using base theme and wish to use B2B module you have to override `theme/modules/ProductView/Synthesis/AddProductToCart.js` from the base theme and provide a way to add to requisition list from the product page (see `<AddProductToRequisitionList>` and `modules/front-commerce-b2b/web/theme/modules/ProductView/Synthesis/AddProductToCart.js` for inspiration). You also need to override `theme/modules/AddToCart/_AddToCart.scss` if you have not already and just copy the styles from the base theme to it (`src/web/theme/modules/AddToCart/_AddToCart.scss`).
```

Comment:
Can you maybe add links to the references?
- `<AddProductToRequisitionList>`
- `modules/front-commerce-b2b/web/theme/modules/ProductView/Synthesis/AddProductToCart.js`

Reply:
0b9365cea93199a19913b3dd32e838f61dc983b9